src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -106,6 +118,23 @@ export default class ResultsViewMutationTable extends MutationTable<IResultsView
         this._columns[MutationTableColumnType.CANCER_TYPE].shouldExclude = ()=>{
             return !this.props.uniqueSampleKeyToTumorType;
         };
+
+        this._columns[MutationTableColumnType.CLONAL].shouldExclude = ()=> {
+            return !this.hasCcfMCopies;
```

Comment:
If `hasCcfMCopies`  is always used in negative form. May want to consider just rename it to `withoutCcfMCopies`